src/components/molecules/artefact-line/artefact-line.jsx

```diff
@@ -8,32 +8,44 @@ import './artefact-line.scss';
 export default ({
   title = '',
   subtitle = '',
-  text = '',
+  date = '',
   to = '',
+  masterData = '',
   imgSrc = '',
   imgAlt = '',
 }) => (
-  <div
-    className="artefact-line"
-    data-component="molecules/artefact-line"
-  >
-    <span>
-      <Link
-        to={ to }
-        triggersInternalTransition={ true }
-        internalTransitionDirection='left'
-      >
-        <Image
-          src={ imgSrc }
-          alt={ imgAlt }
-          additionalClass="-has-box artefact-line-image"
-        />
-      </Link>
-    </span>
-    <span className="line-content">
-      <span className="line-title">{ title }</span>
-      <span className="line-subtitle">{ subtitle }</span>
-      <span className="line-text">{ text }</span>
-    </span>
-  </div>
+    <div
+      className="artefact-line"
+      data-component="molecules/artefact-line"
+    >
+      <div className="artefact-line__image">
+        <Link
+          to={to}
+          triggersInternalTransition={true}
+          internalTransitionDirection='left'
+        >
+          <Image
+            src={imgSrc}
+            alt={imgAlt}
+            modifier="--with-box" // -has-box artefact-line-image
```

Comment:
Sehe gerade erst, dass hier ein Modifier-Klassenteilstring als Parameter reingereicht wird und direkt für die Konstruktion der Klasse verwendet wird. Dabei wird der Style für `image--with-box` im Image-Stylesheet definiert.

Was hältst du davon, wenn man für jeden Modifier einen Boolean-Parameter einführt, um darüber jeweils eine Eigenschaft der Image.Komponente zu steuern. Alternativ auch nur ein Parameter mit sowas in Richtung eines Enums oder Konstanten.
Momentan sehe ich nämlich das Problem, dass man erst in das Komponenten-Stylesheet schauen muss, damit man weiß, welche Modificator es überhaupt für die Komponente gibt.
Fände es besser, wenn die Komponente eine saubere und stabile Schnittstelle bietet.